packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,30 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```typescript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
+const TABLE_PARTITION_KEY: Attribute = { name: 'hashKey', type: AttributeType.String };
+
+const stack = new Stack();
+const globalTables = new GlobalTable(stack, CONSTRUCT_NAME, {
+    dynamoProps: {
+        partitionKey: TABLE_PARTITION_KEY,
+        tableName: TABLE_NAME
+    },
+    regions: [ 'us-east-1', 'us-east-2', 'us-west-2' ]
+});
+```
+
+## Implementation Notes
+Global Dynamodb tables is an odd case currently.  The way this package works -
+
+* Creates a DynamoDB table in a separate stack in each `DynamoDBGlobalStackProps.region` specified
+* Deploys a CFN Custom Resource to your stack's specified region that calls a lambda that runs the aws cli which calls `createGlobalTable()`
+
+### Notes
+
+`DynamoDBGlobalStackProps.dynamoProps` is essentially a copy of `dynamodb.TableProps` - however `tableName` is *required*.
```

Comment:
You can extract all the props besides `tableName` from `TableProps` into ` TableOptions` interface and then have `GlobaProps` and `TableProps` inherit from `Options` (that's kind of how we do this)

Reply:
```ts
// in @aws-cdk/aws-dynamodb

export interface TableOptions {
  // all props besides `tableName`
}

export interface TableProps extends TableOptions {
  tableName?: string;
}

// in aws-dynamodb-global
export interface GlobalStackProps extends TableOptions {
  tableName: string;
}
```